frontend/src/store/modules/applications.js

```diff
@@ -1,26 +1,43 @@
 import ApiService from '../../common/apiService';
+import ApplicationService from '../../common/applicationService';
+import { formatStringToNumericArray } from '../../utils/format';
 import { defineStore } from 'pinia';
-import { SCHOOL_APPLICATION_MOCK } from '../../utils/constants/mocks';
+
+const updateStatusName = (status) => {
+  const newStatus = {
+    'New (Submitted)': 'New',
+    'In Progress (Send Confirmation of Receipt Email)': 'In Progress',
+    '2 - Draft App': 'Draft',
+    '3 - App Submitted': 'Submitted',
+  };
+  return newStatus[status] ? newStatus[status] : status;
+};
 
 export const applicationsStore = defineStore('applications', {
   namespaced: true,
   state: () => ({
     EOIApplications: null,
-    schoolApplicationsMap: new Map(),
+    schoolApplications: null,
+    application: null,
     eoi: null,
     confirmationMessage: null,
   }),
   getters: {
     getConfirmationMessage: (state) => state.confirmationMessage,
     getSchoolApplicationsFormatted: (state) =>
-      // format the data for table view - keys are turned into table headers
-      Object.values(Object.fromEntries(state.schoolApplicationsMap)).map(
-        (v) => ({
-          // TODO: update with id when connected to API
+      // Sort applications by last 4 digits of the applicationnumber
+      state.schoolApplications
+        .sort(
+          (a, b) =>
+            a.iosas_applicationnumber.split('-').pop() -
```

Comment:
Can we use a var name with more than 3 letters?

Reply:
Updated!